dali/base/dautils.cpp

```diff
@@ -3245,9 +3245,25 @@ class CLocalOrDistributedFile: implements ILocalOrDistributedFile, public CInter
                 if (dfile.get())
                     return true;
             }
+
+            StringBuffer dir;
+            StringBuffer cluster;
+#ifdef _CONTAINERIZED
+            // NB: this may work equally as well in bare-metal
```

Comment:
except the default directory in bare metal also includes the component

Reply:
I'll take comment out for now